app/components/certificate/description.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+
+interface DescriptionProps {
+  request: Boolean;
+  validFrom: Date;
+  validTo: Date;
+}
+
+export default function DescriptionSection(props: DescriptionProps) {
+  const { request, validFrom, validTo } = props;
+  let description: String = '';
+
+  switch (request) {
+    case true:
+      description =
+        "Certificate: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s";
+      break;
+    case false:
+      description =
+        "Initial: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s";
+      break;
+  }
+
+  return (
+    <Flex flexDirection="column" gap="3" marginTop="14">
+      <Heading as="h1" size="xl">
+        Certificate
+      </Heading>
+      <Text>{description}</Text>
+      {request && (
```

Comment:
So does `request` here mean "should show this UI"?  If so, name it something like that.